e2e/framework/test_context.go

```diff
@@ -149,11 +150,13 @@ type TestContextType struct {
 	// Wizard specifies whether wizard was used to bootstrap cluster
 	Wizard bool `json:"-" yaml:"-"`
 	// Provisioner defines the type of provisioner to use
-	Provisioner provisionerType `json:"provisioner" yaml:"provisioner" env:"ROBO_PROVISIONER"`
+	Provisioner provisionerType `json:"provisioner" yaml:"provisioner" `
+	// ProvisionTo defines cloud to deploy
+	ProvisionTo string `json:"provision_to" yaml:"provision_to" validate:"omitempty,eq=aws|eq=azure"`
```

Comment:
I would name it `CloudProvider` or something similar to be consistent with the terminology we use in the rest of the codebase (e.g. in gravity).